### PR TITLE
Init russian localization

### PR DESCRIPTION
**Pull request by dianaabv**
Would be awesome to have russian translation especially for students from Central Asia, where russian is the second language. would like to work on that, but help is also welcome.

### CODE REVIEW

**Comment by sftim** [comment]
Добро пожаловать!

Although Prow's listed me as a reviewer, I'm not sure what's involved in adding and approving a new localization.

File: config.toml [hunk 1]
```diff
@@ -203,6 +203,13 @@ time_format_blog = "02.01.2006"
 # A list of language codes to look for untranslated content, ordered from left to right.
 language_alternatives = ["en"]
 
+[languages.ru]
+title = "Kubernetes"
+description = "Production-Grade Container Orchestration"
```
Comment (sftim) [inline config.toml]: Something like:
```suggestion
description = "Контейнерная оркестровка, готовы на производст"
```
maybe?

File: config.toml [hunk 2]
```diff
@@ -203,6 +203,13 @@ time_format_blog = "02.01.2006"
...
+languageName ="Русский"
+weight = 1
+contentDir = "content/ru"
+
```
Comment (sftim) [inline config.toml]: It looks like there should be a `[languages.ru.params]` as well.

File: config.toml (hunk 1 above)
Comment (dianaabv16) [reply config.toml]: I was thinking the same but the description is the same for all languages.

File: config.toml (hunk 2 above)
Comment (dianaabv16) [reply config.toml]: yes, I added in the last commit! thanks

File: config.toml (hunk 1 above)
Comment (sftim) [reply config.toml]: Actually, some localizations do translate this. Check out https://github.com/kubernetes/website/pull/16003/files#diff-618063036395fe9ee107f22b46c9eadeR245

File: README-ru.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+## Running the website locally using Hugo
+ -->
+## 使用 Hugo 在本地运行网站
+
+<!--
+See the [official Hugo documentation](https://gohugo.io/getting-started/installing/) for Hugo installation instructions.
+Make sure to install the Hugo version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L9) file.
+ -->
+См. [Официальную документацию Hugo] (https://gohugo.io/getting-started/install/) для получения инструкций по установке Hugo.
+Обязательно установите версию Hugo, указанную в переменной среды `HUGO_VERSION` в файле [` netlify.toml`] (netlify.toml # L9).
```
Comment (sftim) [inline README-ru.md]:
```suggestion
Обязательно установите версию Hugo, указанную в переменной среды `HUGO_VERSION` в файле [`netlify.toml`](netlify.toml#L9).
```

Beware machine translation!

File: README-ru.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+
+<!--
+Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
+ -->
+Откройте браузер по адресу http://localhost:1313 для просмотра веб-сайта. При внесении изменений в исходные файлы Hugo обновляет веб-сайт и принудительно обновляет браузер.
+
+<!--
+## Running the website locally using Hugo
+ -->
+## 使用 Hugo 在本地运行网站
```
Comment (sftim) [inline README-ru.md]: This doesn't look like Russian to me! :wink:

Comment (dianaabv16) [reply README-ru.md]: oops sorry thanks for that

File: README-ru.md [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+ -->
+
+[![Build Status](https://api.travis-ci.org/kubernetes/website.svg?branch=master)](https://travis-ci.org/kubernetes/website)
+[![GitHub release](https://img.shields.io/github/release/kubernetes/website.svg)](https://github.com/kubernetes/website/releases/latest)
+
+<!--
+Welcome! This repository houses all of the assets required to build the [Kubernetes website and documentation](https://kubernetes.io/).
+We're glad that you want to contribute!
+ -->
+Добро Пожаловать! Эта репозитория содержит все необходимое для того чтобы улучшить [Kubernetes вебсайт и документацию](https://kubernetes.io/)
```
Comment (Potapy4) [inline README-ru.md]: Сейчас это выглядит как машинный перевод. Я бы заменил на:

Добро пожаловать! Этот репозиторий содержит все исходники, необходимые для работы [веб-сайта и документации Kubernetes](https://kubernetes.io)

**Comment by remyleone** [comment]
@dianaabv16 Welcome feel free to reach out on both GitHub and Slack :) http://slack.kubernetes.io/

**Comment by remyleone** [comment]
https://github.com/msheldyakov/website/tree/language_ru/content/ru

**Comment by remyleone** [comment]
@msheldyakov Could you contribute your changes as well?

**Comment by remyleone** [comment]
There is a parallel work that started here #16378 I think merging work would be the way forward

**Comment by remyleone** [comment]
For those interested, a telegram group was started https://t.me/kubernetes_docs_ru

**Comment by sftim** [comment]
/uncc
as I don't speak Russian well enough to comment

**Comment by aisonaku** [comment]
Maybe it is reasonable to close this PR because we agreed to make an initial translation all togethere in this PR https://github.com/kubernetes/website/pull/16404 ?

**Pull request closed without merging**